src/Assets/LightSkyDefense/Scripts/Enemies/EnemySplit.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
+
+public class EnemySplit : MonoBehaviour
+{
+    public GameObject NewEnemy;
+    public int SplitAmount = 2;
+
+    void OnDestroy()
+    {
+        for (int i = 0; i < SplitAmount; i++)
+        {
+            var newEnemy = Instantiate(
+                NewEnemy,
+                transform.position,
+                Random.rotation
+            );
+
+            newEnemy.GetComponent<Damageable>().UpdateHealth(-0.6f);
```

Comment:
We shouldn't update the health on spawn like this since we might want to add an update health effect. We should configure the initial health in the `NewEnemy` game object instead.

Reply:
it took me a while to understand what the hell you meant but I figured it out 😄 . Done